src/main/java/io/confluent/connect/hdfs/wal/WAL.java

```diff
@@ -27,4 +27,5 @@
   void truncate() throws ConnectException;
   void close() throws ConnectException;
   String getLogFile();
+  long getSleepIntervalMs();
```

Comment:
Hmm, a bit of a nit, but since this is only used for testing I'm not sure about making it public for testing. In fact, we separately have some work going on to refactor some of this code to make it more reusable and this seems like an odd addition to the interface given that it's really just public for testing. /cc @kkonstantine 

I'm wondering if just casting to the more specific FSWAL class in the test would be a better solution (and label the FSWAL public API as public for testing)? Keeps the interface clean but allows the test to access the info that it needs.